nix-mode.el

```diff
@@ -622,7 +724,17 @@ The hook `nix-mode-hook' is run when Nix mode is started.
   (setq-local parse-sexp-lookup-properties t)
 
   ;; Automatic indentation [C-j]
-  (setq-local indent-line-function nix-indent-function)
```

Comment:
Keep this line, so nix-indent-function is still used.

Reply:
Done.